src/final2/subtests/InvalidCommandLineArgumentsTest.java

```diff
@@ -12,7 +12,6 @@
  * @author Joshua Gleitze
  * @author Christian Hilden
  * @author Martin Löper
- * @author Roman Langrehr
```

Comment:
Definitely not on purpose! Must have happened while merging, I'm sorry! Put it back in c55208c